### PR TITLE
Issue/6 detail editing

### PR DESCRIPTION
**Pull request by Pergamene**

### CODE REVIEW

File: internal_comps/sp_page/src/components/page-action/sp-page-action-footer.js [hunk 1]
```diff
@@ -96,6 +96,9 @@ export class SpPageActionFooter extends connect(localStore)(LitElement) {
         return LOCALE_EN.SP_PAGE_SELECTION.SELECTIONS.OVERVIEW;
       case 'properties':
         return LOCALE_EN.SP_PAGE_SELECTION.SELECTIONS.PROPERTIES;
+      // @ISSUE: add one for detail
```
Comment (rhyeen) [inline internal_comps/sp_page/src/components/page-action/sp-page-action-footer.js]: Remove comment

File: internal_comps/sp_page/src/components/page-parts/edit/sp-page-title-edit.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
+// @ISSUE: you'll want your own *-edit.js file for sp-page-detail.js
```
Comment (rhyeen) [inline internal_comps/sp_page/src/components/page-parts/edit/sp-page-title-edit.js]: remove comment

File: internal_comps/sp_page/src/components/page-parts/sp-page-overview.js [hunk 3]
```diff
@@ -1,15 +1,15 @@
...
 import '../page-parts/sp-page-summary.js';
 import '../page-parts/edit/sp-page-summary-edit.js';
 import '../page-parts/edit/sp-page-title-edit.js';
 
-import '../options-pane/sp-dropdown-btn';
+import '../options-pane/sp-dropdown-btn.js';
 import { localStore } from '../../state/store.js';
-import { selectPageOverview, PAGE_SELECTION_ACTION_EDIT } from '../../state/action';
-import { Log } from 'interface-handler/src/logger';
+import { selectPageOverview, PAGE_SELECTION_ACTION_EDIT, PAGE_SECTION_TYPE_DETAIL, PAGE_SECTION_TYPE_OVERVIEW } from '../../state/action.js';
```
Comment (rhyeen) [inline internal_comps/sp_page/src/components/page-parts/sp-page-overview.js]: don't need `PAGE_SECTION_TYPE_DETAIL` (I believe)

**Review by rhyeen — CHANGES_REQUESTED** [review]

**Comment by Pergamene** [comment]
image
image

**Review by rhyeen — APPROVED** [review]

**Pull request merged**